src/components/theme/Image/Image.jsx

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import { flattenToAppURL } from '@plone/volto/helpers';
+
+/**
+ * Image component
+ * @param {object | string} image - Plone image as object or url
+ * @param {string} baseUrl - Base URL of the image item (the part before /@@images)
+ * @param {string} alt - Alternative text for the image
+ * @param {boolean} loading - (default: eager) set to `lazy` to lazy load the image
+ */
+export default function Image({
+  image,
+  baseUrl = '',
```

Comment:
I stuck with the image + baseUrl implementation, because it's probably more flexible than having a complete item object. For example, in the view of a custom content type I could have more than one image field and want to show more than one.

Reply:
@pnicolli It makes sense to pass in the specific image that should be rendered, rather than guessing the field name. But I think it is probably a good idea to also include the item anyway, for context in case a customized version of the Image component needs to render the image differently based on some other property of the item.

I will check more carefully whether our current implementation actually depends on this. I think it does but I might be remembering details wrong.